backend/tarsy/agents/parsers/react_parser.py

```
@@ -0,0 +1,439 @@
+        parameters = {}
+        action_input = action_input.strip() if action_input else ""
+        
+        if not action_input:
+            return parameters
+        
+        try:
+            # Try JSON first
+            if action_input.startswith('{'):
+                parameters = json.loads(action_input)
+            else:
+                # Handle YAML-like format: "apiVersion: v1, kind: Namespace, name: superman-dev"
```

Ruff (0.12.2)

Line too long (96 > 88)

(E501)